test/operations/operations.go

```diff
@@ -154,3 +154,22 @@ func configSpecToYaml(configSpec cnao.NetworkAddonsConfigSpec) string {
 	// Note that it is empty otherwise
 	return "Empty Spec"
 }
+
+func LabelConfig(gvk schema.GroupVersionKind, newLabels map[string]string) {
+	By(fmt.Sprintf("Labeling NetworkAddonsConfig with:\n%v", newLabels))
+
+	// Get current Config
+	config := GetConfig(gvk)
+
+	// Label the Config with the desired new labels
+	labels := config.GetLabels()
+	if labels == nil {
+		labels = make(map[string]string)
```

Comment:
if there were not labels before then maybe we can do directly
config.SetLabels(labels) ?

Reply:
I think you mean that if there are no existing labels, then just do `config.SetLabels(newLabels)`
if that's the case then I prefer to have the action (in our case SetLabels) to be done once on all cases (as much as possible)
To have multiple places where the action is happening is more prone to bugs (fix here, but forget to fix the other...)